### PR TITLE
checklist and tabs

### PR DESCRIPTION
**Pull request by rpanai**

### CODE REVIEW

File: pages/examples/checklist-plots.py
Comment (Coding-with-Adam) [inline pages/examples/checklist-plots.py:42]: As per the Readme ([name-of-the-file]-x-), I think the component id should be:
`Output('checklist-plots-x-scatter', 'figure')` And
`Input('checklist-plots-x-checklist', 'value')`

ID's should also be modified in the layout section, per component

File: pages/examples/checklist-plots.py
Comment (Coding-with-Adam) [inline pages/examples/checklist-plots.py:15]: can you tell me more please why you chose to build the fig here as well as in the callback

File: pages/examples/checklist-plots.py
Comment (Coding-with-Adam) [inline pages/examples/checklist-plots.py:2]: This can go on one line:
from dash import Dash, dcc, Input, Output

* no need to import html since we'll be using dbc for the gallery apps

File: pages/checklist-plots.py
Comment (Coding-with-Adam) [inline pages/checklist-plots.py:13]: I'm not 100% sure, but I think you need to add this line at the very end, as in [this example](https://github.com/AnnMarieW/dash-app-gallery/blob/main/pages/axes.py):
```
def layout():
    return example_app(f"pages/examples/{filename}.py")
```

File: pages/checklist-plots.py
Comment (Coding-with-Adam) [inline pages/checklist-plots.py:6]: This should be: `layout_type = "2Row():1Col()",`

File: pages/checklist-plots.py
Comment (Coding-with-Adam) [inline pages/checklist-plots.py:7]: graph="scatter"

File: pages/tabs.py
Comment (Coding-with-Adam) [inline pages/tabs.py:1]: similar comments need to be address, based on the ones form the checklist-plots.py file inside pages

File: pages/tabs.py
Comment (Coding-with-Adam) [inline pages/tabs.py:6]: actually, let's change this to `layout_type = "tabs"`

File: pages/tabs.py
Comment (Coding-with-Adam) [inline pages/tabs.py:10]: let's change this to `callback=None`

File: pages/examples/tabs.py
Comment (Coding-with-Adam) [inline pages/examples/tabs.py:26]: let's use dbc theme please, put everything inside container, and use dbc.Row and dbc.Col instead of html.Div

File: pages/examples/tabs.py
Comment (Coding-with-Adam) [inline pages/examples/tabs.py:1]: no need to import html since we'll be using dbc for the gallery apps

**Review by Coding-with-Adam — CHANGES_REQUESTED** [review]
Thank you for submitting this @rpanai . I've added some suggestions/fixes needed. 

Can you please update the checklist png image so the right sidebar from your computer doesn't show.

File: pages/examples/checklist-plots.py
Comment (rpanai) [reply pages/examples/checklist-plots.py:42]: My bad here.

File: pages/examples/checklist-plots.py
Comment (rpanai) [reply pages/examples/checklist-plots.py:35]: I initially wrote the app on a older version

**Review by Coding-with-Adam — COMMENTED** [review]
this file looks good

**Review by Coding-with-Adam — APPROVED** [review]

**Comment by AnnMarieW** [comment]
Looks great - thanks!  :woman_dancing: 
Thanks @rpanai and @Coding-with-Adam

**Comment by Coding-with-Adam** [comment]
Hi Ann,Can you please revert this merge. Roberto had not finished making all the changes/updates I asked him to make prior to merging.Thank you,Adam SchroederOwnerCharming Data LLC 118 E 124th ST #1808, New York, New York 10035   My Pronouns: He, him, his ¡Hablo español! 
      
        
        
          On Apr 23 2022, at 9:01 pm, Ann Marie Ward ***@***.***> wrote:
          
        
        
          

Looks great - thanks!  💃

Thanks @rpanai and @Coding-with-Adam

**Comment by AnnMarieW** [comment]
Hi Adam - No problem - but it’s probably easier to do a new PR for the changes and I’ll merge those when it’s ready

**Comment by Coding-with-Adam** [comment]
You mean instaed of waiting for him to make the changes, do a PR and have him approve it before merging?

 


  
  
  Adam Schroeder
  Owner
  Charming Data LLC 
  118 E 124th ST #1808, New York, New York 10035 
    
  My Pronouns: He, him, his 
  ¡Hablo español!  
  
  
  
  
 
  
  
    On Mon, Apr 25, 2022, 12:32 am Ann Marie Ward ***@***.***> wrote: 
    
   
   
    
   Hi Adam - No problem - but it’s probably easier to do a new PR for the changes and I’ll merge those when it’s ready 
   —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***> 
   [

{

***@***.***": "http://schema.org",

***@***.***": "EmailMessage",

"potentialAction": {

***@***.***": "ViewAction",

"target": "https://github.com/AnnMarieW/dash-app-gallery/pull/11#issuecomment-1108068416",

"url": "https://github.com/AnnMarieW/dash-app-gallery/pull/11#issuecomment-1108068416",

"name": "View Pull Request"

},

"description": "View this Pull Request on GitHub",

"publisher": {

***@***.***": "Organization",

"name": "GitHub",

"url": "https://github.com"

}

}

]

**Pull request merged**
